java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -53,10 +56,14 @@ public static BaseTransaction<?> transactionTest(BaseTransaction<?> baseTransact
         baseTransaction.hincrBy(key4, field3, 5);
         baseTransaction.hincrByFloat(key4, field3, 5.5);
 
-        baseTransaction.sadd(key5, new String[] {"baz", "foo"});
```

Comment:
we dont need to modify the keys of set commands, it can stay as it was, and list commands will use key6

Reply:
Just following python code here for consistency makes it easier for me to not miss transaction tests and also makes it consistent.